jupinx/theme/minimal/templates/latex.tpl

```diff
@@ -0,0 +1,304 @@
+((*- extends 'article.tplx' -*))
```

Comment:
@AakashGfude is this true?

Reply:
yes. it inherits from some of the base templates of nbconvert.